src/clients/nominatimClient.ts

```
@@ -1,11 +1,14 @@
 import axios from 'axios'
 import type { ReverseGeocodeParams } from '../types/reverseGeocodeTypes.js'
 import type { GeocodeParams } from '../types/geocodeTypes.js'
+import packageJson from '../../package.json' with { type: 'json' };
```

ESLint

[error] Extra semicolon.

(semi)